libmscore/page.cpp

```diff
@@ -399,9 +407,11 @@ void PageFormat::read(XmlReader& e, Score* score)
                         }
                   }
             else if (tag == "page-height")
-                  _size.rheight() = e.readDouble() * 0.5 / PPI;
+                  _size.rheight() = e.readDouble() / SCALE_XML;
             else if (tag == "page-width")
-                  _size.rwidth() = e.readDouble() * .5 / PPI;
+                  _size.rwidth() = e.readDouble() / SCALE_XML;
+            else if (tag == "page-units")
```

Comment:
I'm not sure it's a good idea to store the unit this way. Maybe it's better to save a string representing the unit. @wschweer Any opinion?
Also, we should pick a default, probably the unit used in MuseScore 2 and not store `page-units` at all if the default is used.